### DISCUSSION TITLE
Welcome to shopizer Discussions!

### DISCUSSION THREAD

**Discussion by shopizer-ecommerce** [shopizer-ecommerce/shopizer#672] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by amarnarayan1998** [comment]
Sir I am fresher how to login and clear errors can u explain me plz

**Comment by jaffarali0272** [comment]
Hello Sir, I am trying for elasticsearch feature of shopizer but its not working it return this error.
"com.salesmanager.core.business.exception.ServiceException: Cannot invoke \"com.shopizer.search.autoconfigure.SearchClient.getClient()\" because \"this.searchClient\" is null, Cannot invoke \"com.shopizer.search.autoconfigure.SearchClient.getClient()\" because \"this.searchClient\" is null"

http://localhost:8081/api/v1/search/autocomplete
